examples/with-expo/hooks/useAuth.ts

```diff
@@ -23,7 +24,12 @@ export default function useAuth() {
    */
   const login = async (email: string, password: string) => {
     setLoading(true);
-    await signInWithEmailAndPassword(getAuth(), email, password);
+    await signInWithEmailAndPassword(getAuth(), email, password)
```

Comment:
What about this:

![image](https://github.com/andresjesse/simple-firestore-hooks/assets/2041796/f3caf682-f9cb-428b-aee4-d9a66301917e)

Reply:
This way the error still being throwed and we have a guarantee to complete the loading

Reply:
nice!